pandas/tests/scalar/timedelta/test_timedelta.py

```diff
@@ -376,6 +379,21 @@ def test_unit_parser(self, units, np_unit, wrapper):
             result = Timedelta('2{}'.format(unit))
             assert result == expected
 
+    @pytest.mark.skipif(sys.version_info < (3, 6),
+                        reason="requires python3.6 or higher")
+    @pytest.mark.parametrize('unit', ['Y', 'y', 'M'])
+    def test_unit_m_y_deprecated(self, unit):
+        with tm.assert_produces_warning(FutureWarning,
+                                        check_stacklevel=False) as w1:
```

Comment:
Can you remove the `check_stacklevel=False` here? 
Here it should be work, in the one below for `to_timedelta`, it is fine to keep.